docs/contributing/development_environment.md

```diff
@@ -53,9 +53,37 @@ to create a virtual machine connected to the isolated networks.
 Create the edpm-compute-0 virtual machine.
 ```
 cd install_yamls/devsetup
-make standalone
+make standalone EDPM_COMPUTE_CEPH_ENABLED=false
 ```
 
+Create the edpm-compute-1 virtual machine for an extra Nova compute cell. Note
+that the main cell `nova` (in terms of OSP) is going to become `nova_cell1`,
```

Comment:
@SeanMooney I'm lost a bit with the renaming going on, so am not certain this sentence is correct

Reply:
I think it should be "main ...nova... becomes nova_cell0, and nova_cell0 becomes nova_cell1" ?

Reply:
form the point of view of nova the cell database that was previously call "nova" was the nova cell 1 database

cell0 is a reserved name with special meaning.

its a convention that the cell1 db schma and cell name in the API db should be nova-cell1 or nova_cell1 not nova but tripleo did not follow that convention.
so we are just renaming it to follow best practices

it was always cell one even if it was not named that.